lang/en.js

```diff
@@ -394,7 +393,7 @@ export default {
   },
   "contentfulManual": {
```

Comment:
Is this ever used? I see no reference to it in the codebase.

Reply:
For reference, it's a place for translations in Lokalise which will then be input manually into Contentful, for now.